src/components/Repository.js

```diff
@@ -98,8 +98,11 @@ function Repository({match}) {
       .finally(() => setIsForkLoading(false));
   };
 
+
   const user = getUserFromJwt(Config.auth);
 
+  const showFork = repoOwner !== user.login ? true : isForkLoading;
```

Comment:
@filiptronicek Try this!

```suggestion
  const showFork = repoOwner !== user && user.login ? true : isForkLoading;
```